docs/modules/user-guide/partials/proc_developing-with-devfiles.adoc

```diff
@@ -0,0 +1,36 @@
+[id="proc_developing-with-devfiles_{context}"]
+= Developing with devfiles
+[role="_abstract"]
+
+Develop a node.js “Hello World” application, using the devfile specification. Developing this application introduces you to how a devfile automates and simplifies your development process.
+
+.Prerequisites
+
+* To develop an application by using the devfile specification, you need to be on a cluster. To create a cluster, install link:https://minikube.sigs.k8s.io/docs/start/[minikube start].
+* To access your namespaces, download link:https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/[kubectl].
+* To execute the devfile specification, install the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html/cli_tools/developer-cli-odo#installing-odo[odo cli tool].
+
+.Procedure
+
+. Build your Kubernetes cluster by running `minikube start`.
```

Comment:
I want to keep the steps succinct, but I can provide explanations for the more complex steps if you think that'd be helpful.

Reply:
In general, I'd rather see the steps lead with the command by following the structure of "Run `xyx` to access this functionality" rather than "Access this functionality by running `xyz`." I think it makes the steps more terse, and the important part (the command) is the first thing the user sees. I've edited the steps below with some suggestions along these lines.